src-java/floodlight-service/floodlight-modules/README.md

```diff
@@ -15,3 +15,45 @@ where:
  ratePkts - is the rate in packets for specific type of rule
  disco_packet_size - the size of discovery packet, is being defined in properties file with the key 'org.openkilda.floodlight.switchmanager.SwitchManager.disco-packet-size'.
 ```
+
+#### How to debug Floodlight component
+
+In order to run floodLight in the debug mode we have to add this line of arguments
+```
+"-agentlib:jdwp=transport=dt_socket,address=50506,suspend=n,server=y”
+``` 
+when running the app.
+Particularly in floodLight we can not do this straight to the Dockerfile since it leads us to entrypoint.sh script.
+We have to add these arguments to the
+```
+projectroot/docker/floodlight-modules/entrypoint.sh
+```
+file;
+
+
+in this file find `exec java` line under the ```"$1" = 'floodlight’```  condition.
```

Comment:
a line starting with `exec java`. Seems like there's a missing action here: I found that line and..? 
I'd rewrite lines 27-34 something like: In the ...entrypoint.sh file, in the condition block ```"$1" = 'floodlight’```, add the arguments after the `exec java`, leave the existing arguments as is.

Reply:
thanks, fixed